ludwig/models/trainer.py

```diff
@@ -155,9 +155,9 @@ def __init__(
         :param epochs: Number of epochs the algorithm is intended to be run over
         :type epochs: Integer
         :param learning_rate: Learning rate for the algorithm, represents how
-               much to scale the gradients by
-        :type learning_rate: Integer
-        :param batch_size: Size of batch to pass to the model for training.
+               much to scale the gradients by.  May be 'auto' to auto-tune learning rate.
```

Comment:
Move/copy the comment: "The learning_rate specified in configuration" here.

Does `auto` specify to auto-tune the learning rate, per-epoch? Or to choose a default `target_learning_rate`, or both?